Validation/Geometry/interface/MaterialBudgetData.h

```diff
@@ -56,8 +56,14 @@ class MaterialBudgetData {
     return theH_ScintillatorFractionMB; }
   float getLeadFractionMB()       const {                    
     return theLeadFractionMB; }
-  float getM_NEMA_FR4_plateFractionMB()   const {
-    return theM_NEMA_FR4_plateFractionMB; }
+  float getEpoxyFractionMB()       const {                    
```

Comment:
The distance between parenthesises and const looks a bit too big. Could you please fix this? This applies to other such cases in this file as well.